tests/unit/server/migrations/migrate_v12.test.js

```diff
@@ -0,0 +1,246 @@
+                topic.infoItems.forEach(infoItem =>{
+                    infoItem.details.forEach(checkDetailHasProperty_createdInMinute);
+                    infoItem.details.forEach(checkDetailHasProperty_id);
+                })
+            });
+        });
+
+        it('sets the correct createdInMinute-attribute for all details in minutes and meetingSeries', function() {
+            MigrateV12.up();
+            // detail was created in 1st minute => createdInMinute = 1st Minute
+            expect(firstFakeMinute.topics[0].infoItems[0].details[0].createdInMinute).to.equal(FIRST_MIN_ID);
+            // detail created in 1st minute occured in 2nd Minute => createdInMinute = 1st Minute
+            expect(sndFakeMinute.topics[0].infoItems[0].details[0].createdInMinute).to.equal(FIRST_MIN_ID);
+            expect(fakeMeetingSeries.topics[0].infoItems[0].details[0].createdInMinute).to.equal(FIRST_MIN_ID);
+            expect(fakeMeetingSeries.openTopics[1].infoItems[0].details[0].createdInMinute).to.equal(FIRST_MIN_ID);
+            // detail created in 2nd minute => createdInMinute = 2nd Minute
+            expect(sndFakeMinute.topics[1].infoItems[0].details[0].createdInMinute).to.equal(SND_MIN_ID);
+            expect(fakeMeetingSeries.topics[1].infoItems[0].details[0].createdInMinute).to.equal(SND_MIN_ID);
+            expect(fakeMeetingSeries.openTopics[0].infoItems[0].details[0].createdInMinute).to.equal(SND_MIN_ID);
+            //a new detail was added to an infoItem from an old minute in a new minute => createdInMinute = 2nd Minute
+            expect(sndFakeMinute.topics[0].infoItems[0].details[2].createdInMinute).to.equal(SND_MIN_ID);
+        });
+
+        it('sets the correct _id attribute for all details in minutes and meetingSeries', function() {
+            MigrateV12.up();
+            let detailIdInFirstMinute = firstFakeMinute.topics[0].infoItems[0].details[0]._id;
+            expect(sndFakeMinute.topics[0].infoItems[0].details[0]._id).to.equal(detailIdInFirstMinute);
+            expect(fakeMeetingSeries.topics[0].infoItems[0].details[0]._id).to.equal(detailIdInFirstMinute);
+            expect(fakeMeetingSeries.openTopics[1].infoItems[0].details[0]._id).to.equal(detailIdInFirstMinute);
+        });
+
+    });
+
+    describe('#down', function () {
+
+        beforeEach(function() {
+            let addCreatedInMinuteFakeAttribute = (topic) => {
+                topic.createdInMinute = 'fakeID';
+            };
+            firstFakeMinute.topics.forEach(addCreatedInMinuteFakeAttribute);
+            sndFakeMinute.topics.forEach(addCreatedInMinuteFakeAttribute);
+            fakeMeetingSeries.topics.forEach(addCreatedInMinuteFakeAttribute);
+            fakeMeetingSeries.openTopics.forEach(addCreatedInMinuteFakeAttribute);
+        });
+
+        it('removes the createdInMinute and _id-attribute', function() {
+            MigrateV12.down();
+
+            let checkDetailHasNoAttribute_createdInMinute = detail => {
```

Comment:
Here applies the same as for the the checkDetailHasProperty... helper functions.